src/ngraph/runtime/gpu/gpu_external_function.cpp

```diff
@@ -753,6 +752,10 @@ string runtime::gpu::GPU_ExternalFunction::emit_op_as_function(const Node& node,
     // Work around a compiler warning (*node inside typeid may have effects
     // with shared pointers, which is fine here but clang doesn't like it.)
     auto handler = dispatcher.find(type_index(typeid(node)));
+    if (handler == dispatcher.end())
+    {
+        throw ngraph::unsupported_op(node->description());
```

Comment:
I guess in `emit_op_as_function`, `node` is a const ref., so we need `node.description()` here.